pkg/controller/install/openshift/openshift.go

```diff
@@ -178,6 +240,39 @@ func egress(u *unstructured.Unstructured) *unstructured.Unstructured {
 	return u
 }
 
+func ensureOpenShiftRegistryForConfigMap(u *unstructured.Unstructured) *unstructured.Unstructured {
+	if u.GetKind() == "ConfigMap" && u.GetName() == "config-deployment" {
+		data := map[string]string{"registriesSkippingTagResolving": "ko.local,dev.local,docker-registry.default.svc:5000,image-registry.openshift-image-registry.svc:5000"}
```

Comment:
We should not be editing the list of tag resolving registries here. Doing so actually makes the SSL related changes irrelevant. Instead, we want the SSL changes in this PR but should not touch this config setting at all with our operator.